boot.py

```diff
@@ -1,3 +1,12 @@
+import sublime
+
+if int(sublime.version()) > 4000:
+    sublime.error_message(
+        """The installed version of LSP package is not compatible with Sublime Text 4.\n"""
```

Comment:
The installed -> The currently installed
... of LSP package ... -> ... of the LSP package ...